packages/pubsub/system-test/pubsub.js

```diff
@@ -513,7 +513,6 @@ describe('pubsub', function() {
         assert.ifError(err);
 
         assert.deepEqual(policy.bindings, []);
-        assert.strictEqual(policy.etag, 'ACAB');
```

Comment:
What happens now?

Reply:
It returns a Buffer, I tried decoding using the generated Policy object but that returns a ByteBuffer from protobufjs.